doc/nrf/libraries.rst

```diff
@@ -33,6 +33,14 @@ Here you can find documentation for these libraries, including API documentation
    ../../include/net/*
 
 
+.. toctree::
+   :maxdepth: 1
+   :caption: Binary libraries:
+   :glob:
+
+   ../../lib/bin/lwm2m_carrier/*
```

Comment:
This will put all rst files inside lwm2m_carrier on the same level, including the changelog.
A quick fix is to change it to `../../lib/bin/lwm2m_carrier/lwm2m_carrier.rst`. A better way would be to include `../../lib/bin/*/*` and move all other files to a doc/ subfolder.

Reply:
you're right, I'll fix